src/js/components/productSupplier/ProductSupplierListTable.jsx

```diff
@@ -30,6 +32,11 @@ const ProductSupplierListTable = ({ filterParams }) => {
     exportProductSuppliers,
   } = useProductSupplierActions({ fireFetchData });
 
+  const isUserAdminWithProductManager = useUserHasPermissions({
```

Comment:
Let's change it to the suggested convention `canManageProducts` and let's stick to that version of variable names moving forward.